DatadogObjc/Sources/RUM/RUM+objc.swift

```diff
@@ -649,17 +649,30 @@ public class DDRUMMonitor: NSObject {
 
     @objc
     public func addAttribute(
-        forKey key: String,
+        forKey key: AttributeKey,
         value: Any
     ) {
         swiftRUMMonitor.addAttribute(forKey: key, value: AnyEncodable(value))
     }
 
     @objc
-    public func removeAttribute(forKey key: String) {
+    public func addAttributes(_ attributes: [AttributeKey: Any]) {
+        let attributes = attributes.reduce(into: [AttributeKey: AnyEncodable]()) {
+            $0[$1.key] = AnyEncodable($1.value)
+        }
+        swiftRUMMonitor.addAttributes(attributes)
```

Comment:
/nit You have an extension for this:
```suggestion
        swiftRUMMonitor.addAttributes(attributes.dd.swiftAttributes)
```

Reply:
👌